tests/backend/core/Core/Functional/QueryProcedureTest.php

```diff
@@ -215,7 +214,7 @@ public function testInterfaceFiltersWithoutScope(): void
         static::assertCount(1, $availableFiltersDefault);
     }
 
-    //#######################
```

Comment:
???

Reply:
what's gone is gone. it had no value except visual disturbance anyway.